package.json

```diff
@@ -29,8 +29,8 @@
     "test": "jest",
     "test:coverage": "jest --coverage",
     "test:watch": "jest --watch",
-    "licenseCheck:prepare": "license-tool/build.sh",
-    "licenseCheck:run": "license-tool/run.sh"
+    "dash-licenses:check": "docker run -ti -v $(pwd):/workspace/project quay.io/che-incubator/dash-licenses:next --check",
+    "dash-licenses:update": "docker run -ti -v $(pwd):/workspace/project quay.io/che-incubator/dash-licenses:next:next"
```

Comment:
```suggestion
    "dash-licenses:update": "docker run -ti -v $(pwd):/workspace/project quay.io/che-incubator/dash"
```

Reply:
shouldn't we leave at least one `next` tag?